Strategy Tutorials/Tutorial03 Pairs Trading - Copula vs Cointegration.html

```diff
@@ -165,14 +165,14 @@
 </tbody>
 </table>
 </div>
-<pre class="prettyprint linenums">def _parameter(self, family, tau):   	  	
```

Comment:
Try and avoid white space changes. These just add to noise and don't actually change the code result